package.json

```diff
@@ -53,7 +53,10 @@
     "url-loader": "^0.5.7",
     "urthecast": "^1.0.0",
     "webpack": "^1.12.14",
-    "worker-loader": "^0.7.0"
+    "worker-loader": "^0.7.0",
+    "pbf": "^1.3.2",
+    "vector-tile": "1.2.0",
+    "point-in-polygon": "1.0.0"
```

Comment:
[point-in-polygon](https://github.com/substack/point-in-polygon) doesn't seem to have a licence in the repo, it just says MIT. Can I just write:
```
### point-in-polygon

> MIT licence
```

Or do I have to write the licence for him?